about_page.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width initial-scale=1.0" />
+        <title>About: InnovationX Africa Summit</title>
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.css"/>
+        <link rel="stylesheet" href="./style/style.css" />
+    </head>
+    <body>
+    <section class="container">
+        <header>
+            <div class="top-navigation">
+                <div class="media-links">
+                    <ul>
+                        <li><a><i class="fa fa-facebook" aria-hidden="true"></i></a></li>
+                        <li><a><i class="fa fa-twitter" aria-hidden="true"></i></a></li>
+                        <li><a>English</a></li>
+                        <li><a>My Page</a></li>
+                        <li><a>Logout</a></li>
+                    </ul>
+                </div>
+                <div>
+                    <li class="logo-link"><a href="index.html"><img class="summitlogo" src="./images/logos/CSi.png" alt="Summit Logo"/></a>
+                        <ul>
+                            <li><a href="about_page.html" id="about-active"> About</a></li>
+                            <li><a href="#"> Program</a></li>
+                            <li><a href="#"> Join</a></li>
+                            <li><a href="#"> Sponsor</a></li>
+                            <li><a href="#"> News</a></li>
+                            <li class="campaign"><a href="#"> IX Campaign</a></li>
+                        </ul>
+                </div>
+            </div>
+            <nav class="toggle-menu">
+                <div class="mobile-menu">
+                    <a href="index.html"><img class="mobile-logo" src="./images/logos/CSi.png" alt="Summit Logo"/></a>
+                </div>
+                <div class="nav-links">
+                    <i class="fa fa-times fa-2x" aria-hidden="true"></i>
+                    <ul>
+                        <li><a href="about_page.html" id="about-active">About</a></li>
+                        <li><a href="#">Progress</a></li>
+                        <li><a href="#">Join</a></li>
+                        <li><a href="#">Sponsor</a></li>
+                        <li><a href="#">News</a></li>
+                    </ul>
+                </div>
+                <div class="campaign">
+                    <ul>
+                        <li>
+                            <a href="#">Campaign</a>
+                        </li>
+                    </ul>
+                </div>
+            </nav>
+        </header>
```

Comment:
### CHANGES FOR THE HOME AND ABOUT PAGE


- Hey, great job, however, let's improve our semantic usage on our webpage by moving the header out the 
`<section class="container">` as the header shouldn't be wrapped in this section tag. It's meant to be a standalone element in your webpage,
- Also, the header is the first element in your webpage that holds your introductory content, and the nav tag which the list of your navigation items. please, fix this. :pray: